twitter-client.js

```diff
@@ -101,19 +99,29 @@ Twitter.prototype.postFollow = function (params, error, success) {
 
 Twitter.prototype.doRequest = function (url, error, success) {
   url = formatUrl(url);
-
-  this.oauth.get(url, this.accessToken, this.accessTokenSecret, (err, bod, res) => {
-    if(!err && res.statusCode == 200) {
-      limits = {
-        "x-rate-limit-limit": res.headers['x-rate-limit-limit'],
-        "x-rate-limit-remaining": res.headers['x-rate-limit-remaining'],
-        "x-rate-limit-reset": res.headers['x-rate-limit-reset']
-      };
-      success(JSON.parse(bod));
-    } else {
-      console.error('do request error' + err);
+  var arrData = [];
```

Comment:
Ah, good call. I used arrData since the doRequest function may be used later on to get more than just user objects, but that's all I'll be working with for now.